src/OnboardingSPA/components/Drawer/DrawerPanel/DesignColors.js

```diff
@@ -328,7 +335,7 @@ const DesignColors = () => {
 						<div
 							className="color-palette__colors--tertiary"
```

Comment:
Should we rename all of these variables and classes to `header-background` instead of `tertiary`?